src/Data/These.purs

```diff
@@ -98,6 +102,7 @@ instance showThese :: (Show a, Show b) => Show (These a b) where
   show (That y) = "(That " <> show y <> ")"
   show (Both x y) = "(Both " <> show x <> " " <> show y <> ")"
 
+-- | Collapse a `These` value into single value.
```

Comment:
Perhaps also include "given functions to handle each constructor"? I'm thinking something a little bit less verbose than:

https://github.com/purescript/purescript-either/blob/v4.1.1/src/Data/Either.purs#L238-L238

though that does nicely provide a usage example.

Reply:
Yeah, that sounds good.

Reply:
It's basically a more concise pattern match function.